test/hotspot/jtreg/serviceability/jvmti/SuspendWithObjectMonitorWait/SuspendWithObjectMonitorWait.java

```diff
@@ -87,6 +107,7 @@ public class SuspendWithObjectMonitorWait {
     native static int wait4ContendedEnter(SuspendWithObjectMonitorWaitWorker thr);
 
     public static void main(String[] args) throws Exception {
+        int test = Integer.parseInt(args[0]);
```

Comment:
I think this will throw if the test is called without any parameters.

Also, the `usage` function needs to be updated to reflect the addition of the
test number parameter.